frontend/public/components/marketplace/kubernetes-marketplace.jsx

```diff
@@ -89,7 +116,7 @@ export class MarketplaceListPage extends React.Component {
   render() {
     const {loaded, loadError} = this.props;
     const {items} = this.state;
-    return <StatusBox data={items} loaded={loaded} loadError={loadError} label="Resources">
+    return <StatusBox data={items} loaded={loaded} loadError={loadError} label="Resources" EmptyMsg={'Please check that the marketplace operator is running. If you are using your own quay.io appregistry, please ensure your operators are properly documented. For more information visit https://github.com/operator-framework/operator-marketplace'} >
```

Comment:
`EmptyMsg={() => <MsgBox title="No Marketplace Items Found" detail="Please check that the marketplace operator is running. If you are using your own quay.io appregistry, please ensure your operators are properly documented. For more information visit https://github.com/operator-framework/operator-marketplace" />}
`